src/network/bridge.rs

```diff
@@ -494,6 +498,17 @@ fn create_interfaces(
                     InfoKind::Bridge,
                 );
                 create_link_opts.mtu = data.mtu;
+
+                if data.vrf.is_some() {
+                    let vrf = match host.get_link(netlink::LinkID::Name(
+                        data.vrf.as_ref().unwrap().to_string(),
```

Comment:
should this be handled `unwrap().to_string()` ? What would happen here is `unwrap()` fails ? There is no wrapping of error here.

Reply:
The best way to write this is `if let Some(vrf) = data.vrf {` then in the if block you can directly access the vrf var.

Reply:
Thank you very much for your input, I don't have much experience  with rust. I rewrote this part in my latest commit, could you check if it is better?